recipes/graylog-forwarder/files/graylog-forwarder.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=Graylog Cloud Forwarder
+Documentation=http://docs.graylog.org/
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+User=graylog-forwarder
+Group=graylog-forwarder
+Restart=on-failure
+RestartSec=10
+LimitNOFILE=64000
+
+WorkingDirectory=/usr/share/graylog-forwarder
```

Comment:
Let's change this to the `data_dir` (`/var/lib/graylog-forwarder`) to avoid writing anything to `/usr/share/graylog-forwarder`.

Reply:
This is fixed.